test/run-test.ps1

```diff
@@ -63,7 +63,10 @@ Get-ChildItem -Recurse -Filter Dockerfile |
         $buildImage = "sdk-build-$appName"
         $dotnetNewParam = ""
         if ($sdkTag -like "*1.1-sdk-msbuild*") {
-            $dotnetNewParam = "-t Console1.1"
+            $dotnetNewParam = "console --Framework netcoreapp1.1"
+        }
+        elseif ($sdkTag -like "*1.0-sdk-msbuild*") {
+            $dotnetNewParam = "console --Framework netcoreapp1.0"
```

Comment:
Consider factoring out the commonality between these two values of `dotnetNewParam`, as the only difference now is "1.1" vs. "1.0" at the end.